packages/js/product-editor/src/blocks/generic/text-area/editor.scss

```diff
@@ -0,0 +1,32 @@
+.wp-block-woocommerce-product-text-area-field {	
+	.rich-text {
+		width: 100%;
+		min-height: calc($gap-larger * 3);
+		background-color: $white;
+		box-sizing: border-box;
+		border: 1px solid #757575;
```

Comment:
Hardcoded color. It's a shame we don't have a variable for this.

Reply:
Creating a new var, at least for now, sounds good for you?

Reply:
https://github.com/woocommerce/woocommerce/pull/44151/commits/ad88986c3013b664082a39fedcb1aa50f712dd31 Here we are using the `$gray-700` var.